easybuild/easyconfigs/l/LncLOOM/LncLOOM-2.0-foss-2020b.eb

```diff
@@ -0,0 +1,72 @@
+easyblock = 'PythonBundle'
+
+name = 'LncLOOM'
+version = '2.0'
+
+homepage = 'https://github.com/lncLOOM/lncLOOMv2/'
+description = """LncLOOM is a graph-based framework that uses integer
+programming to identify combinations of short motifs that are deeply conserved
+in rapidly evolving sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('BLAST+', '2.11.0'),
+    ('Gurobi', '9.1.0', '', True),
+    ('Biopython', '1.76'),  # setup.py <= 1.76
```

Comment:
This is a problem, because it'll make this `LncLOOM` installation incompatible with other `foss/2020b` modules that depend on BioPython.

Can you check if Biopython 1.76 is really needed, or if they're just being conservative here?

It's very likely the latter, I suspect just using Biopython 1.78 instead will work just fine...

Reply:
It seems that the reason is that newer Biopython versions dropped support for Py2. Given that I'm building with 3.8 anyway, I'll bump and check if it works but don't expect any real problems, there was just one breaking change in the changelog and it doesn't touch LncLOOM code at all.